Assets/MixedRealityToolkit/InputSystem/Pointers/GenericPointer.cs

```diff
@@ -92,13 +106,13 @@ public virtual void OnPreRaycast()
             Ray pointingRay;
             if (TryGetPointingRay(out pointingRay))
             {
-                Rays[0].CopyRay(pointingRay, (PointerExtent ?? InputSystem.FocusProvider.GlobalPointingExtent));
```

Comment:
What happened to the `GlobalPointingExtent`? Would be nice to have an option to not have to set the same extent on every pointer.

Reply:
It's still there. We set it when we create the pointer now.